src/OPL-UK-3.0.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="OPL-UK-3.0" name="United
+   Kingdom Open Parliament Licence v3.0" listVersionAdded="">
+      <crossRefs>
+         <crossRef>https://www.parliament.uk/site-information/copyright-parliament/open-parliament-licence/</crossRef>
+      </crossRefs>
+      <text>
+         <titleText>
+            United Kingdom Open Parliament Licence v3.0
+         </titleText>
+         <p>
+            Open Parliament Licence
+         </p>
+         <p>
+            You are encouraged to use and re-use the information that
+            is available under this licence freely and flexibly, with
+            only a few conditions. Using information under this licence
+         </p>
+         <p>
+            Use of copyright and database right material made
+            available under this licence (the ‘information’) indicates
+            your acceptance of the terms and conditions below.
+         </p>
+         <p>
+            The Licensor grants you a worldwide, royalty-free,
+            perpetual, non-exclusive licence to use the
+            information subject to the conditions below.
+         </p>
+         <p>
+            This licence does not affect your freedom under
+            fair dealing or fair use or any other copyright
+            or database right exceptions and limitations.
+         </p>
+         <p>
+            You are free to:
+         </p>
+         <list>
+            <item>
+               <bullet>*</bullet>
+               <p>
+                  copy, publish, distribute and transmit the information
+                  adapt the information exploit the information
+                  commercially and non-commercially, for example,
+                  by combining it with other information, or by
+                  including it in your own product or application
+               </p>
```

Comment:
looking at how the license displays in its original form, there should be 3 list items (i.e., the lines starting with "copy...", "adapt..", and "exploit..." as you see https://www.parliament.uk/site-information/copyright-parliament/open-parliament-licence/